component/site/views/default/list/tmpl/csvprintevents.php

```diff
@@ -24,10 +26,15 @@
 
 $fields = array();
 
+$app = JFactory::getApplication();
+$currentMenuId = JSite::getMenu()->getActive()->id;
+$mparams   = $app->getMenu()->getItem($currentMenuId)->params;
+
```

Comment:
Humm, I thought and tested that and didn't think it worked. I'll run some tests today and you can re-review by end of the day :-)